app/models/location_polygon.rb

```diff
@@ -1,8 +1,8 @@
 class LocationPolygon < ApplicationRecord
   scope :cities, -> { where(location_type: "cities") }
   scope :counties, -> { where(location_type: "counties") }
-  scope :london_boroughs, -> { where(location_type: "london_boroughs") }
   scope :regions, -> { where(location_type: "regions") }
+  scope :other, -> { where(location_type: "other") }
```

Comment:
I think it might pay to keep the location type of `london_boroughs`, as well as an `other` type

Reply:
The reason I got rid of it is that some London boroughs are named after towns they contain, and I thought it was preferable to have those places come up under 'Towns and cities' rather than nowhere. As I don't want to deal with giving places multiple types I got rid of the london_borough type, as it's not being used.

Reply:
Yeah forget this lol I thought we still had these on the home page 😆